lib/Reconciler.lua

```diff
@@ -44,6 +44,20 @@ local function isPortal(element)
 	return element.component == Core.Portal
 end
 
+--[[
+	Sets the value of a reference to a new rendered object.
+	Correctly handles both function-style and object-style refs.
+]]
+local function applyRef(ref, newRbx)
+	if not ref then return end
```

Comment:
Let's explicitly check `if ref == nil` and then put the return statement on its own line.

Reply:
Fixed now!